duijf-julia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:edge
+MAINTAINER robin@kingsquare.nl
+
+RUN apk add --no-cache julia --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
+
+# Create working dir
+RUN mkdir -p /var/app
+COPY ./raffler.jl /var/app
```

Comment:
I assume the newer docker versions actually support copying a file to a dir while the older versions expect the full path?